coeus-impl/src/main/java/org/kuali/coeus/sys/framework/controller/KcTransactionalDocumentActionBase.java

```diff
@@ -613,8 +614,18 @@ protected void initialDocumentSave(KualiDocumentFormBase form) throws Exception
      * @return the status (INITIATED, SAVED, etc.)
      */
     private String getDocumentStatus(Document doc) {
-        return doc.getDocumentHeader().getWorkflowDocument().getStatus().getLabel();
-    }
+    	try {
+    		return doc.getDocumentHeader().getWorkflowDocument().getStatus().getLabel();
+    	} catch (RiceRuntimeException e) {
+    		LOG.error("Could not find doc.getDocumentNumber(): " + doc.getDocumentNumber());
```

Comment:
Please combine into a single log statements so the logging messages don't get interleaved.